src/website/views/index.html

```diff
@@ -0,0 +1,8 @@
+<!doctype html>
+<html>
+<head>
+<meta http-equiv="refresh" content="0; url=/getting-started">
```

Comment:
@jllr Can we leverage the [metalsmith-alias](https://github.com/fortes/metalsmith-alias) plugin for redirects?